datapackage.json

```diff
@@ -1,116 +1,80 @@
-      "mediatype": "text/csv",
       "name": "annual",
       "path": "data/annual.csv",
+      "format": "csv",
+      "mediatype": "text/csv",
       "schema": {
         "fields": [
           {
             "name": "Source",
             "type": "string"
           },
           {
-            "description": "YYYY",
             "name": "Year",
-            "type": "year"
+            "type": "date",
+            "description": "YYYY"
           },
           {
-            "description": "Average global mean temperature anomalies in degrees Celsius relative to a base period. GISTEMP base period: 1951-1980. GCAG base period: 20th century average.",
             "name": "Mean",
-            "type": "number"
+            "type": "number",
+            "description": "Average global mean temperature anomalies in degrees Celsius relative to a base period. GISTEMP base period: 1951-1980. GCAG base period: 20th century average."
           }
         ]
       }
     },
     {
-      "format": "csv",
-      "mediatype": "text/csv",
       "name": "monthly",
       "path": "data/monthly.csv",
+      "format": "csv",
+      "mediatype": "text/csv",
       "schema": {
         "fields": [
           {
             "name": "Source",
             "type": "string"
           },
           {
-            "description": "YYYY-MM",
-            "format": "any",
             "name": "Date",
-            "type": "date"
+            "type": "date",
+            "description": "YYYY-MM"
           },
           {
-            "description": "Monthly mean temperature anomalies in degrees Celsius relative to a base period. GISTEMP base period: 1951-1980. GCAG base period: 20th century average.",
             "name": "Mean",
-            "type": "number"
+            "type": "number",
+            "description": "Monthly mean temperature anomalies in degrees Celsius relative to a base period. GISTEMP base period: 1951-1980. GCAG base period: 20th century average."
           }
         ]
       }
     }
   ],
-  "sources": [
-    {
-      "name": "GISTEMP Global Land-Ocean Temperature Index",
-      "path": "http://data.giss.nasa.gov/gistemp",
-      "title": "GISTEMP Global Land-Ocean Temperature Index"
-    },
-    {
-      "name": "Global component of Climate at a Glance (GCAG)",
-      "path": "http://www.ncdc.noaa.gov/cag/time-series/global",
-      "title": "Global component of Climate at a Glance (GCAG)"
-    }
-  ],
-  "title": "Global Temperature Time Series",
-  "version": "0.1.0",
   "views": [
     {
-      "name": "graph",
-      "title": "Monthly mean temperature anomalies in degrees Celsius relative to a base period",
-      "specType": "simple",
-      "resources": ["annual", "monthly"],
-      "spec": {
-        "type": "column",
+      "id": "graph",
+      "label": "Graph",
+      "type": "Graph",
+      "state": {
         "group": "Year",
         "series": [
           "Mean"
-        ]
+        ],
+        "graphType": "columns"
```

Comment:
@gradedSystem I'm not sure if we have this type and generally if this is correct spec. I would rather copy the old view spec.